### PR TITLE
Don't build for tags

### PR DESCRIPTION
**Pull request by hzoo**
sounds good? should do this everywhere

### CODE REVIEW

**Comment by danez** [comment]
Why?

**Comment by hzoo** [comment]
Do you use the emails? slack pings build status already, and tag ci isn't necessary since it already builds to master

**Comment by danez** [comment]
I was never pinged by slack before. For me emails is the only way currently to get notified if my PRs fail, but if I can somehow setup slack I'm happy with that.

File: .travis.yml [hunk 1]
```diff
@@ -51,4 +51,9 @@ after_success:
   - 'if [ "$JOB" = "test-coverage" ]; then bash <(curl -s https://codecov.io/bash) -f coverage/coverage-final.json -F babylon ; fi'
 
 notifications:
+  email: false
```
Comment (jridgewell) [inline .travis.yml]: 👎

**Comment by hzoo** [comment]
K i'l remove the email thing

**Pull request merged**